### PR TITLE
Update example code for refactor

### PR DESCRIPTION
**Pull request by jjbrosnan**
Also clean up code syntax

### CODE REVIEW

File: TickingHeartRate/runTickingHeartRateReplay.py [hunk 1]
```diff
@@ -1,34 +1,35 @@
-from deephaven.TableTools import readHeaderlessCsv
+from deephaven import csv
```
Comment (jakemulf) [inline TickingHeartRate/runTickingHeartRateReplay.py:1]: I don't think this is the right import. Shouldn't it be

```suggestion
from deephaven import read_csv
```

File: TickingHeartRate/runTickingHeartRateReplay.py [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+    for x in range(1, csv_files):
+        next_file = ("/data/examples/TickingHeartRate/csv/%d.csv" % x)
+        print(next_file)
+        path = pathlib.Path(next_file)
         if path.exists() and path.is_file():
-            nextHR=readHeaderlessCsv(nextFile).update("Timestamp=Column1", "Heart_rate=Column2").select("Timestamp", "Heart_rate")
-            nextRecord=nextHR.getRecord(0, "Timestamp", "Heart_rate")
-            timestamp=DBTimeUtils.autoEpochToTime(nextRecord[0])
-            hrTableWriter.logRow(timestamp, int(nextRecord[1]))
+            next_hr = csv.read(next_file, headless = True).update("Timestamp=Column1", "Heart_rate=Column2").select("Timestamp", "Heart_rate")
```
Comment (jakemulf) [inline TickingHeartRate/runTickingHeartRateReplay.py:26]:
```suggestion
            next_hr = read_csv(next_file, headless = True).update("Timestamp=Column1", "Heart_rate=Column2").select("Timestamp", "Heart_rate")
```

**Review by jakemulf — CHANGES_REQUESTED** [review]

File: TickingHeartRate/runTickingHeartRateReplay.py (hunk 1 above)
Comment (chipkent) [reply TickingHeartRate/runTickingHeartRateReplay.py:1]: I think either syntax is ok.

File: TickingHeartRate/runTickingHeartRateReplay.py (hunk 2 above)
Comment (chipkent) [reply TickingHeartRate/runTickingHeartRateReplay.py:26]: I think either syntax is ok.

Comment (chipkent) [reply TickingHeartRate/runTickingHeartRateReplay.py:26]: `.update("Timestamp=Column1", "Heart_rate=Column2").select("Timestamp", "Heart_rate")` can just be written more efficiently and compactly as `.view("Timestamp=Column1", "Heart_rate=Column2")`.  This does no allocations.  The original version does 4 column allocations.

File: TickingHeartRate/runTickingHeartRateReplay.py [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
 from deephaven import DynamicTableWriter, Types as dht
-from deephaven import DBTimeUtils
-from deephaven.DBTimeUtils import autoEpochToTime
+from deephaven import DateTimeUtils
 import pathlib
 import time
 import threading
 
+import sys
+
```
Comment (chipkent) [inline TickingHeartRate/runTickingHeartRateReplay.py:9]:
```suggestion
```
Does not appear to be used.

**Review by chipkent — CHANGES_REQUESTED** [review]

File: Fit/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Fit file as available from Strava
+Heart_rateHeart_rate# Fit file as available from Strava
```
Comment (chipkent) [inline Fit/README.md:1]:
```suggestion
# Fit file as available from Strava
```

File: Fit/accessFit.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-from deephaven import DynamicTableWriter, Types as dht
+Heart_rateHeart_rateHeart_ratefrom deephaven import DynamicTableWriter, Types as dht
```
Comment (chipkent) [inline Fit/accessFit.py:1]:
```suggestion
from deephaven import DynamicTableWriter, Types as dht
```

File: MetricCentury/README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Metric Century 
+Heart_rateHeart_rate# Metric Century
```
Comment (chipkent) [inline MetricCentury/README.md:1]:
```suggestion
# Metric Century 
```

File: README.md [hunk 7]
```diff
@@ -24,7 +24,7 @@ The following folders can be found in this repository:
 - **[`Taxi`](./Taxi)** - Yellow Taxi trip records.
 - **[`TensorFlow`](./TensorFlow)** - Statistically calculate positive/negative sentiment using machine-learning
   training mechanisms based on an RSS feed from Seeking Alpha.
-- **[`TickingHeartRate`](./TickingHeartRate)** - Simulated ticking heart rate data.
+- **[`TickingHeartRate`](./TickingHeartRate)** - Simulated ticking Heart_rate data.
```
Comment (chipkent) [inline README.md:27]:
```suggestion
- **[`TickingHeartRate`](./TickingHeartRate)** - Simulated ticking heart rate data.
```

File: TickingHeartRate/README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Ticking Heart Rate in Deephaven
+Heart_rateHeart_rate# Ticking Heart Rate in Deephaven
```
Comment (chipkent) [inline TickingHeartRate/README.md:1]:
```suggestion
# Ticking Heart Rate in Deephaven
```

File: TickingHeartRate/runTickingHeartRateReplay.py [hunk 9]
```diff
@@ -1,17 +1,15 @@
...
 import threading
 
-import sys
-
 # Max number of csv files to pull in
 csv_files=500
 
-# Setup deephaven tables to hold Heart Rate results
-column_names = ["Timestamp", "Heart Rate"]
+# Setup deephaven tables to hold Heart_rate results
```
Comment (chipkent) [inline TickingHeartRate/runTickingHeartRateReplay.py:11]:
```suggestion
# Setup deephaven tables to hold heart rate results
```

File: TickingHeartRate/runTickingHeartRateReplay.py [hunk 10]
```diff
@@ -1,17 +1,15 @@
...
+column_names = ["Timestamp", "Heart_rate"]
```
Comment (chipkent) [inline TickingHeartRate/runTickingHeartRateReplay.py:12]:
```suggestion
column_names = ["Timestamp", "HeartRate"]
```
Naming convention for column names is camel case

File: TickingHeartRate/runTickingHeartRateReplay.py [hunk 11]
```diff
@@ -23,7 +21,7 @@ def thread_func():
         print(next_file)
         path = pathlib.Path(next_file)
         if path.exists() and path.is_file():
-            next_hr = csv.read(next_file, headless = True).update("Timestamp=Column1", "Heart_rate=Column2").select("Timestamp", "Heart_rate")
+            next_hr = read_csv(next_file, headless = True).update("Timestamp=Column1", "Heart_rate=Column2").select("Timestamp", "Heart_rate")
```
Comment (chipkent) [inline TickingHeartRate/runTickingHeartRateReplay.py:24]:
```suggestion
            next_hr = read_csv(next_file, headless = True).view("Timestamp=Column1", "HeartRate=Column2")
```

File: Fit/accessFit.py [hunk 12]
```diff
@@ -1,36 +1,39 @@
...
-print("Number of data points:" + str(len(records)))
+print("Number of data points: {}".format(len(records)))
 
 # Setup deephaven tables to hold results
 # Heart rate
-columnNames = ["Timestamp", "Heart Rate"]
-columnTypes = [dht.datetime, dht.int_]
-hrTableWriter = DynamicTableWriter(columnNames, columnTypes)
-heartRateData = hrTableWriter.getTable()
+column_names = ["Timestamp", "Heart Rate"]
```
Comment (chipkent) [inline Fit/accessFit.py:20]: Column names have spaces.  They should be camel case with a caps at the start and no spaces.

File: Fit/accessFit.py [hunk 13]
```diff
@@ -1,36 +1,39 @@
...
+column_types = [dht.datetime, dht.int_]
+hr_table_writer = DynamicTableWriter(column_names, column_types)
+heart_rate_data = hrTableWriter.getTable()
 # Gps data
-columnNames = ["Timestamp", "Enhanced Altitude m", "Enhanced Speed m/s", "GPPS Accuracy m", "Position lat semicircles", "Position long semicircles", "Speed m/s"]
-columnTypes = [dht.datetime, dht.double, dht.double, dht.int_, dht.int_, dht.int_, dht.double]
-gpsTableWriter = DynamicTableWriter(columnNames, columnTypes)
-gpsData = gpsTableWriter.getTable()
+column_names = ["Timestamp", "Enhanced Altitude m", "Enhanced Speed m/s", "GPPS Accuracy m", "Position lat semicircles", "Position long semicircles", "Speed m/s"]
```
Comment (chipkent) [inline Fit/accessFit.py:25]: camel case.  These also have non supported characters like `/`

**Review by chipkent — CHANGES_REQUESTED** [review]

**Review by chipkent — APPROVED** [review]

**Review by hythloda — APPROVED** [review]

**Review by jakemulf — APPROVED** [review]

**Pull request merged**